django/santropolFeast/santropolFeast/settings.py

```diff
@@ -135,13 +135,11 @@
 
 USE_TZ = True
 
-# not import the real utils here! We don't want an infiny loop
-gettext = lambda x: x
 
 # List of supported languages
 LANGUAGES = (
-    ('fr', gettext('French')),
-    ('en', gettext('English')),
+    ('fr', 'French'),
```

Comment:
I agree that it should be done this way. However this is not how how it was done: the `gettext` was only a `lambda x: x` definition. I removed it because it did not add anything.

I will add a commit that put the strings in the original languages.